### DISCUSSION TITLE
How are unknown browser versions handled?

### DISCUSSION THREAD

**Discussion by wilx** [babel/babel#12384] [Q&A]
(I am opening this as a discussion item because I am not sure if this is actually a defect or not. Related to https://github.com/browserslist/browserslist/issues/556)

I am observing odd behaviour with my compilation. If I use this, everything seems to be OK, normal, no polyfills are used:

```
  "browserslist": {
    "production": [
      "Firefox >= 83",
      "Chrome >= 86"
    ]
  }
```

But if I raise the version of Chrome to 87 like this, I suddenly get a lot of polyfills:

```
  "browserslist": {
    "production": [
      "Firefox >= 83",
      "Chrome >= 87"
    ]
  }
```

Is Chrome 87 really that bad or is it just that it is unknown version and it defensively uses polyfills?

I have Chrome "Version 87.0.4280.66 (Official Build) (64-bit)".

**Comment by nicolo-ribaudo** [comment]
We mark features as, for example, "supported starting from chrome 54": even if there is a new version, this logic should automatically handle them.

I know that there was a bug in a recent chrome version and thus the `array.reduce` polyfill is needed, maybe it's that? You can try passing `debug: true` to `@babel/preset-env`'s option with the two different `browserslist` configs to see exactly which are the differences.

**Comment by wilx** [comment]
I am attaching two logs, one when I specify Chrome 86, the current state of the project, and another log where I specify Chrome 87. It is this project: https://github.com/wilx/sane-twitch-chat/

- [chrome86.txt](https://github.com/babel/babel/files/5581583/chrome86.txt)
- [chrome87.txt](https://github.com/babel/babel/files/5581584/chrome87.txt)

**Reply by nicolo-ribaudo** [reply]
It looks like Babel is getting an empty browsers list from Browserslist. Could you try updating Browserslist itself to the last version?

**Reply by wilx** [reply]
I have `browserslist@4.14.7` here. I believe it is the latest version.

**Reply by wilx** [reply]
And, if it matters, `caniuse-lite@^1.0.30001157`.
